flink-connector/src/main/java/io/delta/flink/sink/committer/AbstractDeltaGlobalCommitter.java

```diff
@@ -0,0 +1,12 @@
+package io.delta.flink.sink.committer;
+
+
+import io.delta.flink.sink.committables.AbstractDeltaCommittable;
+import io.delta.flink.sink.committables.AbstractDeltaGlobalCommittable;
+import org.apache.flink.api.connector.sink.GlobalCommitter;
+
+/**
+ * Marker interface for internal class.
```

Comment:
What does "Marker" mean? Just interface should be fine. And make it clear in the docs that this is not meant for public use.

Reply:
Marker interface is actually a common java term, https://www.baeldung.com/java-marker-interfaces

Reply:
@tdas changed to `Marker interface for internal class, not meant for public use.`
this `marker` term does seem in place (as followed for instance by example link provided by @scottsand-db )